data_science_colleges.csv

```diff
@@ -243,6 +247,7 @@ Rutgers University,Online Master of Information,M,T,F,US,NJ,Camden,School of Com
 Rutgers University,Master of Business And Science Degree in Analytics - Discovery Informatics & Data Sciences,M,F,T,US,NJ,Camden,Division of Continuous Education and Outreach,http://psm.rutgers.edu/programs/analytics-data-sciences
 Rutgers University,Post MBA Certificate in Analytics and Information Management,C,F,T,US,NJ,Camden,Rutgers Business School,http://www.business.rutgers.edu/mba/admissions/post-mba-certificate
 Rutgers University,MBA with Analytics and Information Management Concentration,M,F,T,US,NJ,Camden,Rutgers Business School,http://www.business.rutgers.edu/mba/concentrations/aim
+Rutgers University, Online MBA, M,T,F,US,NJ,Camden,Rutgers Business School,http://online.rutgers.edu/master-business-admin/
```

Comment:
in order to be included, the MBA must have a focus or emphasis on business analytics (one than 1 analytics course)